<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menuRecent">
     <property name="title">
      <string>最近打开</string>
     </property>
     <addaction name="actionNone"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="menuRecent"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFindNext"/>
    <addaction name="actionReplace"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>导航</string>
    </property>
    <addaction name="actionForward"/>
    <addaction name="actionBackword"/>
    <addaction name="actionLocation"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>工具</string>
    </property>
    <widget class="QMenu" name="menuView">
     <property name="title">
      <string>界面可见性</string>
     </property>
     <addaction name="actionView"/>
     <addaction name="actionAtrtri"/>
     <addaction name="actionAsiss"/>
     <addaction name="actionToolbar"/>
     <addaction name="actionStatus"/>
    </widget>
    <addaction name="menuView"/>
    <addaction name="actionColor"/>
    <addaction name="actionCite"/>
    <addaction name="actionCheck"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>重复</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>查找</string>
   </property>
  </action>
  <action name="actionFindNext">
   <property name="text">
    <string>查找下一个</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>替换</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="text">
    <string>前进</string>
   </property>
  </action>
  <action name="actionBackword">
   <property name="text">
    <string>后退</string>
   </property>
  </action>
  <action name="actionLocation">
   <property name="text">
    <string>定位</string>
   </property>
  </action>
  <action name="action_23">
   <property name="text">
    <string>界面可见性控制</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>文本颜色</string>
   </property>
  </action>
  <action name="actionCite">
   <property name="text">
    <string>被引用实例</string>
   </property>
  </action>
  <action name="actionCheck">
   <property name="text">
    <string>合法性校验</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>视图区</string>
   </property>
  </action>
  <action name="actionAtrtri">
   <property name="text">
    <string>属性区</string>
   </property>
  </action>
  <action name="actionAsiss">
   <property name="text">
    <string>辅助区</string>
   </property>
  </action>
  <action name="actionToolbar">
   <property name="text">
    <string>工具栏</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="text">
    <string>状态栏</string>
   </property>
  </action>
  <action name="actionNone">
   <property name="text">
    <string>None</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
